docs/src/content/docs/pt-br/guides/components.mdx

```diff
@@ -169,13 +218,200 @@ import { LinkCard } from '@astrojs/starlight/components';
 />
 
 <CardGrid>
-	<LinkCard
-		title="Escrevendo Markdown"
-		href="/pt-br/guides/authoring-content/"
-	/>
+	<LinkCard title="Escrevendo Markdown" href="/pt-br/guides/authoring-content/" />
 	<LinkCard title="Componentes" href="/pt-br/guides/components/" />
 </CardGrid>
 
+### Asides
```

Comment:
We don't have a great word for aside in Portuguese, but since there is even the html tag `<aside>` I thought it would be more understandable to leave it in english. It even matches the translator decision on the ["Authoring Content in Markdown" translation](https://starlight.astro.build/pt-br/guides/authoring-content/#asides) to leave it as is.